Geocoding.md

```diff
@@ -66,6 +66,8 @@ Each element of the returned **results** array will be formatted as follows:
 | sublocality | The neighborhood, borough, township, etc. for the address. Sublocalities are typically more specific than cities and may be returned even if the locality field is empty. **This property may be absent**. |
 | landmark | The full name of the landmark (such as a military base or island) returned by the service. **This property may be absent**. |
 | viewport | A bounding box describing a rectangle that encloses the location. The viewport field contains two coordinate objects -- **northeast** and **southwest** -- each with **lat** and **lng** decimal values. It also contains a **suggested_radius** decimal field that contains the distance in miles from the center of the viewport to a corner. **This property and its elements are always present and non-empty.** |
+| metropolitan_statistical_area | The metropolitan statistical area in which the location is. Metropolitan statistical areas are specific to the US. **This property may be absent**. |
```

Comment:
maybe phrase this as "The metropolitan statistical area that contains this location"

As well, it probably wouldn't hurt to throw in wikipedia links explaining what MSA's and DMA's are since there may be customers who aren't familiar with them but could still extract value from them